src/pages/Chat/Chat/index.tsx

```diff
@@ -99,8 +162,31 @@ export default (): JSX.Element => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  // Initial load messages when chatRef is set
   useEffect(() => loadMessages(), [loadMessages, chatRef]);
 
+  // Get real time chat updates
+  useEffect(() => {
+    if (!chatRef) return () => null;
+
+    firstLoad.current = true;
```

Comment:
Ok, stupid question: if firstLoad is initialized as true, is this statement really necessary?

Reply:
Changed